{"name": "Проверка https://dev-api.getprime.pro/api/v1/auth - invalid-api-key", "status": "passed", "description": "Негативная проверка с передачей неверного ключа ", "steps": [{"name": "Проверка https://dev-api.getprime.pro/api/v1/auth на код 401", "status": "passed", "start": 1753178913443, "stop": 1753178913647}, {"name": "Проверка текста message. Ожидали текст 'Вы не авторизованы. Пожалуйста, выполните вход и повторите попытку.' Получили 'Вы не авторизованы. Пожалуйста, выполните вход и повторите попытку.'", "status": "passed", "start": 1753178913892, "stop": 1753178913892}], "start": 1753178913442, "stop": 1753178913892, "uuid": "e9893f94-f4ef-44fe-a498-5d9f8a37925e", "historyId": "f6aad7747424500f0f55fb65296e0431", "testCaseId": "f6aad7747424500f0f55fb65296e0431", "fullName": "tests.test_api.TestApiGet#test_auth_wrong_key", "labels": [{"name": "tag", "value": "apitest"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApiGet"}, {"name": "host", "value": "Robotek3000"}, {"name": "thread", "value": "3780-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}